delete-draft-publication-handler/src/test/java/no/unit/nva/publication/delete/DeleteDraftPublicationHandlerTest.java

```diff
@@ -0,0 +1,129 @@
+package no.unit.nva.publication.delete;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.PublicationStatus;
+import no.unit.nva.publication.PublicationGenerator;
+import no.unit.nva.publication.exception.NotFoundException;
+import no.unit.nva.publication.service.PublicationService;
+import no.unit.nva.publication.service.PublicationsDynamoDBLocal;
+import no.unit.nva.publication.service.impl.DynamoDBPublicationService;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JsonUtils;
+import org.junit.Rule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
+import org.mockito.Mockito;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.util.UUID;
+
+import static nva.commons.handlers.ApiGatewayHandler.ALLOWED_ORIGIN_ENV;
+import static nva.commons.utils.IoUtils.inputStreamFromResources;
+import static nva.commons.utils.IoUtils.streamToString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@EnableRuleMigrationSupport
+public class DeleteDraftPublicationHandlerTest {
```

Comment:
What is the expected behavior when a publication has a draft doi and is drafted for deletion? Let's say there was a failure in deleting the draft doi

Reply:
This is a legal state until the deletion process has completed.

Reply:
But what is the handler's response?